curriculum/js1/6.js

```diff
@@ -5,8 +5,15 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, possibleDivisor = a -1, sum = 0) => {
+  if (possibleDivisor === 1) {
+    return sum;
+  }
```

Comment:
This looks good, my only recommendation is to put it all on one line like this to clean up the presentation/readability:
```javascript
if (possibleDivisor === 1) return sum;
```